src/test/java/core/basesyntax/service/impl/ParserImplTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParserImplTest {
+    @Test
+    public void parseTransaction_Ok() {
+        List<String> lines = List.of("type,fruit,quantity",
+                "b,banana,20",
+                "p,lemon,10",
+                "s,berry,15");
+        List<Transaction> actual = List.of(new Transaction("b", new Fruit("banana"), 20),
+                new Transaction("p", new Fruit("lemon"), 10),
+                new Transaction("s", new Fruit("berry"), 15));
+        List<Transaction> expected = new ParserImpl().parse(lines);
```

Comment:
i guess here u shoud change naming - 'actual' will be 'expected' and 'expected' will be 'actual', cause for expected u doing .parse, but u testing at least it and it shoud be actual)